extensions/amp-instagram/0.1/amp-instagram.js

```diff
@@ -85,8 +86,10 @@ class AmpInstagram extends AMP.BaseElement {
     const image = this.win.document.createElement('amp-img');
     // This will redirect to the image URL. By experimentation this is
     // always the same URL that is actually used inside of the embed.
-    image.setAttribute('src', 'https://www.instagram.com/p/' +
-        encodeURIComponent(this.shortcode_) + '/media/?size=l');
+    const imgSrc = resolveUrlAttr('amp-img', 'src',
```

Comment:
It should be internal only anyway.

On Tue, Aug 30, 2016 at 5:51 PM, Malte Ubl <malte.ubl@gmail.com> wrote:

> SGTM
>
> On Tue, Aug 30, 2016 at 5:22 PM, Dima Voytenko <notifications@github.com>
> wrote:
>
>> In extensions/amp-instagram/0.1/amp-instagram.js
>> <https://github.com/ampproject/amphtml/pull/4700#discussion_r76904683>:
>>
>> > @@ -85,8 +86,10 @@ class AmpInstagram extends AMP.BaseElement {
>> >      const image = this.win.document.createElement('amp-img');
>> >      // This will redirect to the image URL. By experimentation this is
>> >      // always the same URL that is actually used inside of the embed.
>> > -    image.setAttribute('src', 'https://www.instagram.com/p/' +
>> > -        encodeURIComponent(this.shortcode_) + '/media/?size=l');
>> > +    const imgSrc = resolveUrlAttr('amp-img', 'src',
>>
>> Ideally we should do it via property. These calls are made often and
>> reading an attribute is pretty slow.
>>
>> —
>> You are receiving this because you commented.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/ampproject/amphtml/pull/4700/files/1aa0d710d3dccdfc767a4c932c5020abf0660c6e#r76904683>,
>> or mute the thread
>> <https://github.com/notifications/unsubscribe-auth/AAFeT7p6vZ5D9vuqPnsNYHIBllEE-hFFks5qlMkygaJpZM4JskC5>
>> .
>>
>
>